specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2016-09-01-preview/examples/SearchServiceGetServiceStatistics.json

```diff
@@ -0,0 +1,43 @@
+{
+    "parameters": {
+        "api-version": "2016-09-01-Preview"
```

Comment:
@natinimni could you provide also `searchServiceName` parameter?

Reply:
Similarly to all other Azure Search APIs, 'searchServiceName' is taken from the URL path (not a query parameters)...

Reply:
Slight correction to what @natinimni said -- The search service name is in the host name, not the URL path (each tenant in Azure Search has their own unique endpoint). I'm not aware of any way to specify such parameters in the example files.

Reply:
@natinimni @brjohnstmsft I understand that this parameter is translated to a host name but it's still a parameter and it has to be in examples. AFAIK, there are no exceptions by how the parameter is presented in wire.

Reply:
For example
```json
    "parameters": {
        "api-version": "2016-09-01-Preview",
        "searchServiceName": "somevalue"
    },
```

Reply:
Thanks @sergey-shandar - I've added searchServiceName to all of out API examples, including this one.